files/scponly-chroot.sh

```diff
@@ -0,0 +1,43 @@
+#! /bin/bash
+
+ALTROOT="$1"; shift
+BINARIES="$@"
+
+for b in $BINARIES
+do
+    d=$(dirname "${b}")
+    if [ ! -d "${ALTROOT}${d}" ]; then
+        /bin/mkdir -p "${ALTROOT}${d}"
+    fi
+    /bin/cp "${b}" "${ALTROOT}${d}"
+
+    LIBS=$(ldd "${b}" | awk '{ print $3 }' | egrep -v -v ^'\(')
+
+    for i in ${LIBS}; do
+        d=$(dirname "${i}")
+        if [ ! -d "${ALTROOT}${d}" ]; then
+            /bin/mkdir -p "${ALTROOT}${d}"
+        fi
+            /bin/cp "${i}" "${ALTROOT}${d}"
+    done
+
+    ld=$(ldd "${b}" | grep 'ld-linux' | awk '{ print $1 }')
+    d=$(dirname "${ld}")
+
+    if [ ! -d "${ALTROOT}${d}" ]; then
+        /bin/mkdir -p "${ALTROOT}${d}"
+    fi
+    /bin/cp "${ld}" "${ALTROOT}${d}"
+
+done
+
+/bin/cp "/lib64/libnss_* ${ALTROOT}/lib64/"
+
+if [ ! -d "${ALTROOT}/dev" ]; then
```

Comment:
I don't think this is really needed as it's fairly safe to go ahead and create this directory if it isn't already there. This script only runs once.

Reply:
I put those guards in place because if another chrooted user is created with the same altroot it won't redo what's been done. I guess it doesn't matter but it just seemed like good practice

Reply:
I think we guard that elsewhere in the resource.

Reply:
Please go ahead and still remove this.